transactions/0061.md

```diff
@@ -0,0 +1,174 @@
+# BRC-61: Compound Merkle Path Format
```

Comment:
Is this at all related to the bode that has already been implemented in `go-bc`? https://github.com/libsv/go-bc/blob/master/merkleproof.go

Reply:
The existing implementation is for individual proofs for one txid. This allows the storage of multiple without repeating internal hashes.

That's one distinction. The other is that there is no extraneous info. This would be the minimum data required to be able to validate. The existing proof formats all have extra data which is nice to have but not necessary.

Reply:
For example we could have paths for 1,000 transactions all in one block. The idea being a merkle service would be asked to share the paths for all of them in one api request.

The compound proof would reduce the number of internal branch hashes depending on the proximity of the transactions in the block. Rather than having to repeat hashes, it would only ever share a subset of the hashes required to cover every path.